circuits/scripts/downloadTestCircuits.sh

```diff
@@ -7,6 +7,6 @@ mkdir -p build/test/
 cd build/test
 
 wget --quiet -O circuits.tar.gz \
-    https://macitestcircuits.blob.core.windows.net/circom2/macitestcircuits.tar.gz
+    https://maci-develop-fra.s3.eu-central-1.amazonaws.com/v1.1.2/maci_keys_10-2-1-2_dev.tar.gz
```

Comment:
Why do test circuits change?

Reply:
tbf I think we can remove this altogether as we have the same script in integrationTests which is what we actually use. This script is not used and the link was outdated (pre v1.1.1)